client/Dockerfile

```diff
@@ -1,11 +1,11 @@
 FROM node:8
 
-ADD yarn.lock /yarn.lock
 ADD package.json /package.json
+ADD package-lock.json /package-lock.json
```

Comment:
Tu préfères npm ?
Perso, je suis plus fan de yarn que de npm, je les trouve meilleurs en termes de perf et messages dans la console.

Reply:
Oui, je préfère npm.
Il est quasiment aussi rapide maintenant (il a rattrapé son retard)
Il a quelque features en moins mais d'autres cool en plus (ex: auto-completion des taches que tu peux runner)
Ca reste le gestionnaire de package officiel pour Node, installé par défaut, et je suis pour sa standardisation au lieu d'une multiplication des outils a utiliser
https://blog.risingstack.com/yarn-vs-npm-node-js-package-managers/
https://iamturns.com/yarn-vs-npm-2018/
https://medium.com/wemake-services/is-yarn-still-a-thing-3c6886410c83

Tu peux aussi en discuter avec Rémy Luciani, il a également de bon arguments pour npm.